frontend/src/App.tsx

```diff
@@ -0,0 +1,58 @@
+import { Grid, GridItem } from '@chakra-ui/react';
+import { useSocketIOListeners } from './app/socket';
+import ImageRoll from './features/gallery/ImageRoll';
+import CurrentImage from './features/gallery/CurrentImage';
+import LogViewer from './features/system/LogViewer';
+import Settings from './features/sd/Settings';
+import PromptInput from './features/sd/PromptInput';
+import InitImage from './features/sd/InitImage';
+import ProgressBar from './features/header/ProgressBar';
+import SiteHeader from './features/header/SiteHeader';
+import Variant from './features/sd/Variant';
+
+const App = () => {
+    useSocketIOListeners();
+    return (
+        <>
+            <Grid
+                width='100vw'
+                height='100vh'
+                templateAreas={`
+                    "progressBar progressBar progressBar"
+                    "header header header"
+                    "menu prompt imageRoll"
+                    "menu variant imageRoll"
+                    "menu currentImage imageRoll"`}
+                gridTemplateRows={'4px 40px 100px 32px auto'}
+                gridTemplateColumns={'300px auto 388px'}
+                gap='2'
+            >
+                <GridItem area={'progressBar'}>
+                    <ProgressBar />
+                </GridItem>
+                <GridItem pl='2' pr='2' area={'header'}>
+                    <SiteHeader />
+                </GridItem>
+                <GridItem pl='2' area={'menu'} overflowY='scroll'>
```

Comment:
On devices with visible scrollbars, this container is overflowing horizontally because of the added width of the vertical scrollbar.
![image](https://user-images.githubusercontent.com/2250252/189541006-d58f5b2e-8488-4312-979c-cf0e17a36d0c.png)